lib/sqlalchemy/sql/selectable.py

```diff
@@ -1952,6 +1957,14 @@ def __init__(self, name, *columns):
         for c in columns:
             self.append_column(c)
 
+        schema = kw.pop("schema", None)
+        if schema is not None:
+            self.schema = schema
+        if kw:
+            raise exc.ArgumentError(
+                "Unsupported argument(s): %s" % ",".join(kw)
```

Comment:
here it's probably best to use `"Unsupported argument(s): %s" % list(kw)` because if `kw` has non `str` keys the `join` method will raise

Reply:
I suppose if somewhere else in the init `kw` is used, you could get non-string arguments - fixed, committed, and pushed